sync/class.jetpack-sync-module-callables.php

```diff
@@ -77,6 +77,7 @@ public function enqueue_full_sync_actions() {
 		 * @param boolean Whether to expand callables (should always be true)
 		 */
 		do_action( 'jetpack_full_sync_callables', true );
+		remove_action( 'jetpack_sync_before_send', array( $this, 'maybe_sync_callables' ) );
```

Comment:
shouldn't we call this before `do_action( 'jetpack_full_sync_callables', true );` ?